### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by elenapetra**
endpoints:
  GET - https://zq3zdypgs7.execute-api.eu-west-1.amazonaws.com/dev/products
  GET - https://zq3zdypgs7.execute-api.eu-west-1.amazonaws.com/dev/products/{productId}
Link to FE PR: https://github.com/elenapetra/cloud-tasks/pull/2

### CODE REVIEW

File: product-service/swagger.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+        default:
+          description: Unexpected error
+      requestBody:
```
Comment (dav-dio) [inline product-service/swagger.yml]: GET requests cannot have a requestBody.

Please paste your Swagger file into https://editor.swagger.io/ and check for errors before submitting.

File: product-service/serverless.yml
Comment (dav-dio) [inline product-service/serverless.yml:1]: Please remove the comments in this file. You can save the patterns somewhere for reference, but don't keep them in the codebase.

**Review by dav-dio — CHANGES_REQUESTED** [review]

File: product-service/swagger.yml (hunk 1 above)
Comment (elenapetra) [reply product-service/swagger.yml]: I've updated the swagger, fixing errors.

File: product-service/serverless.yml
Comment (elenapetra) [reply product-service/serverless.yml:1]: I've removed the comments from the .yml file.

**Review by dav-dio — APPROVED** [review]

**Pull request merged**